src/actions/event-actions.js

```diff
@@ -547,11 +547,11 @@ export const getEvents =
 
     const params = {
       expand:
-        "speakers,type,created_by,track,sponsors,selection_plan,location,tags,media_uploads,media_uploads.media_upload_type,actions,actions.type",
+        "speakers,type,created_by,track,sponsors,selection_plan,location,tags,media_uploads,media_uploads.media_upload_type,actions",
       relations:
```

Comment:
review following relations
  "feedback": [],
            "current_attendance": [],
            "slides": [],
"public_comments": [],
"links": [],
"videos": [],
"media_uploads": [],
"extra_questions": [],
"actions": [],
i dont think they are needed at presentation level

Reply:
also review sponsors:[] relations
its retrieving a lot of unneeded data ( you probably just need id and name )
 "sponsors": [
                {
                    "id": 5,
                    "name": "Meta",
                    "sponsorships": [
                        1311,
                        1307,
                        1312,
                        1310,
                        1313,
                        1956,
                        1775,
                        2118,
                        1864,
                        1735,
                        1901,
                        1736,
                        2085,
                        1865,
                        2096,
                        1772,
                        1782,
                        1954,
                        1871,
                        1785,
                        1873,
                        1847,
                        1791,
                        1870,
                        1866,
                        1835,
                        1908,
                        1839,
                        1921,
                        1860,
                        1923,
                        1796,
                        1913,
                        1914,
                        1789,
                        1964,
                        1779,
                        1786,
                        1992,
                        1841,
                        1665,
                        1721,
                        1880,
                        2093,
                        1829,
                        1776,
                        1741,
                        1931,
                        1962,
                        3208,
                        3207,
                        3093,
                        3066,
                        3560,
                        4332,
                        4456,
                        4307,
                        5029,
                        6025,
                        6054
                    ],
                    "project_sponsorships": []
                }
            ],